### DISCUSSION TITLE
Plans to update with the official release of Svelte 5?

### DISCUSSION THREAD

**Discussion by caidanw** [spinspire/pocketbase-sveltekit-starter#33] [General]
Hey there, I've been looking at this starter project for use in a new project. I love what's been done so far and think there is a ton of value here.

With the official release of Svelte 5, I checked this repo to see if it had been updated to support the official release (over the pre-release that is currently targeted).

I know I can manually update the dependency myself, but I wanted to start a discussion about updating this repo to include the new official version by default.

Are there currently any plans to update the repo?

**Comment by jkdoshi** [comment]
Thank you for raising this. The short answer is yes.

I had been neck deep in another project until yesterday. But now I have a bit more time to work on this.

**Reply by keesfluitman** [reply]
nice project man. learning from it! Pocketbase also had some breaking changes. So this project is still stuck on 0.22 i believe. Otherwise, really smart hooks, pocketbase utils etc. The watch function caught my attention and I learned a lot.

**Reply by jkdoshi** [reply]
Again yes. I want to upgrade both Svelte and PocketBase. Just not getting the time to do it.

**Comment by jkdoshi** [comment]
Please see the latest version. Both are done.
